components/card/index.en-US.md

```diff
@@ -0,0 +1,24 @@
+---
+category: Components
+type: Presentation
+title: Card
+---
+
+Common card container.
+
+## When to use
+
+The most basic card container. You can use it to contain text, lists, pictures and paragraphs.
+
+## API
+
+```html
+<Card title="Card title">Card content</Card>
+```
+
+| Property     | Description           | Type     | Default       |
+|----------|----------------|----------|--------------|
+| title    | Card title | React.Element   |  -  |
+| extra    | Operating area on the top right corner of the card | React.Element   | - |
```

Comment:
I missed it. We've used "action area" instead of "operating area" in one of the other examples. For the sake of coherence we should use either "corner content" or "action area" in all places.